cody/cody.md

```diff
@@ -0,0 +1,8 @@
+Hello my name is Cody
```

Comment:
If we look at the initial requirements for this section

```md
Write a few paragraphs about what you are hoping to get out of your work experience while working with the developement team OR a few paragraphs about version control and its pro's and cons.
Both options will require a small amount of planning
Option 1 will not require research so you will be able to focus efforts on your commit history
Option 2 will require some research but the code review will allow a peer to confirm understanding
```

I would expect a little more detail to what you have experienced and what you were hoping to learn. Could you update and add a paragraph for each day of this week and describe details about who/what you have done